test/petab/test_petab_import_pysb.py

```diff
@@ -3,22 +3,27 @@
 """
 
 import os
-
 import numpy as np
-import petabtests
 import yaml
+import logging
 
 import pypesto.optimize as optimize
 from pypesto.petab.pysb_importer import PetabImporterPysb
 # must import after previous, otherwise circular import issues :(
 from amici.petab_import_pysb import PysbPetabProblem
+import petabtests
+
+# In CI, bionetgen is install here
+BNGPATH = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..', '..', 'BioNetGen-2.3.2'))
+if 'BNGPATH' not in os.environ:
+    logging.warning(f"Env var BNGPATH was not set. Setting to {BNGPATH}")
+    os.environ['BNGPATH'] = BNGPATH
```

Comment:
Not optimal for people who might have BNG on `PATH` but no `BNGPATH` set. But not sure how many people that will affect... So probably okay.

Reply:
true ... well, easy to change if it's a problem, leaving for the moment.